lib/router.js

```diff
@@ -19,7 +19,7 @@ Router.prototype.methods = ['get', 'post', 'put', 'delete', 'head',
 Router.prototype.add = function r_add(method, path, handler) {
   method = method.toLowerCase();
   if (this.methods.indexOf(method) === -1) {
-    throw new Error('Method %s is not supported', method);
```

Comment:
I tried `new Error('Method', method, 'is not supported')`, the only exception message I have is 'Method'. @lodr are you sure it's the way to go?